.github/workflows/update-labels.yml

```diff
@@ -28,3 +30,33 @@ jobs:
               repo: context.repo.repo,
               name: "wait for update"
             })
+
+  pr_commented:
+    if: github.event.issue && github.event.issue.pull_request && github.event.comment.user.login == github.event.issue.user.login && (contains(github.event.issue.labels.*.name, 'wait for update') || contains(github.event.issue.labels.*.name, 'discuss') || contains(github.event.issue.labels.*.name, 'need test cases')) && !contains(github.event.issue.labels.*.name, 'user responded')
+    runs-on: ubuntu-latest
+    steps:
+      - name: update label when user responds
+        uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.issues.addLabels({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              labels: ["user responded"]
+            })
+
```

Comment:
`removeLabel` to remove "wait for update" label is missing. 😬 

ref:
https://github.com/monkeyDluffy6017/apisix/blob/d643de37c2701861fbcf4775d0dcc0d049954fb9/.github/workflows/update-labels.yml#L27-L32